resources/erlang_zypper_repository_on_suse.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+#
+# Cookbook Name:: rabbitmq
+# Resource:: erlang_zypper_repository_on_suse
```

Comment:
Never mind my earlier comment. I suggest that we rename this to `erlang_zypper_repository_on_suse_factory` or even just `erlang_zypper_repository`? For other RPMs I can imagine alternative locations such as GitHub or may even PackageCloud one day (they have their own Chef cookbook, though).

Would it make sense to expect more than one package location or be extra specific that we use SUSE Factory here?

Reply:
Will rename in `erlang_zypper_repository_on_suse_factory`.
We have to use the Factory repositories in Leap distros to use le last RabbitMQ and Erlang versions